xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/v3/dto/marketdata/OkexDepth.java

```diff
@@ -0,0 +1,58 @@
+package org.knowm.xchange.okcoin.v3.dto.marketdata;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigDecimal;
+import java.text.ParseException;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import java.util.function.BiConsumer;
+import org.knowm.xchange.okcoin.OkexExchangeV3;
+
+public class OkexDepth {
+
+  public final SortedMap<BigDecimal, BigDecimal> bids;
+  public final SortedMap<BigDecimal, BigDecimal> asks;
+  private Date timestamp;
+
+  public OkexDepth(
+      @JsonProperty("bids") List<Object[]> bidsJson,
```

Comment:
create proper DTOs instead of Object[]

you can have DTOs in Jackson that parse an array JSON type